distribution/ddf/pom.xml

```diff
@@ -30,7 +30,7 @@
     <packaging>pom</packaging>
     <properties>
         <distribution.file.name>ddf-${project.version}</distribution.file.name>
-        <opendj-embedded.app.version>1.3.1</opendj-embedded.app.version>
+        <opendj-embedded.app.version>1.3.2-SNAPSHOT</opendj-embedded.app.version>
```

Comment:
Since we are getting close to a 2.9.0 release, we should make sure opendj is released as well.  Are we close to an opendj release?